cfme/intelligence/reports/reports.py

```diff
@@ -152,6 +165,18 @@ class ReportDetailsView(CloudIntelReportsView):
     @View.nested
     class report_info(WaitTab):  # noqa
         TAB_NAME = "Report Info"
+        # Keeping `group_title` empty since the summary form has no title
+        report_id = SummaryFormItem("", "ID")
+        title = SummaryFormItem("", "Title")
+        primary_filter = SummaryFormItem("", "Primary (Record) Filter")
+        secondary_filter = SummaryFormItem("", "Secondary (Display) Filter")
+        sort_by = SummaryFormItem("", "Sort By")
+        based_on = SummaryFormItem("", "Based On")
+        user = SummaryFormItem("", "User")
+        group = SummaryFormItem("", "EVM Group")
+        updated_on = SummaryFormItem("", "Updated On")
```

Comment:
Q: why we are not using `SummaryForm` instate of `SummaryformItem`? It will reduce that much declaration.

Reply:
Hi @digitronik, there is no group_title to pass to SummaryForm, so I had to individually define each item with SummaryFormItem.

Reply:
ok :+1:  other option is parametrization but its fine

Reply:
I'm curious about the other implementation, can you elaborate?

Reply:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/automate/buttons.py#L108
here repetition of key value pair.